src/clr-angular/data/datagrid/datagrid-column.spec.ts

```diff
@@ -25,6 +23,7 @@ import { StateDebouncer } from './providers/state-debouncer.provider';
 import { TableSizeService } from './providers/table-size.service';
 import { DomAdapter } from '../../utils/dom-adapter/dom-adapter';
 import { DatagridRenderOrganizer } from './render/render-organizer';
+import { ColumnsService } from './providers/columns.service';
 
 const PROVIDERS_NEEDED = [
```

Comment:
Can we just reuse the same providers list for this test too?

Reply:
Done